packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1232,10 +1237,18 @@ export default class Meetings extends WebexPlugin {
         !(err instanceof PasswordError) &&
         !(err instanceof PermissionError)
       ) {
-        // if there is no meeting info we assume its a 1:1 call or wireless share
         LoggerProxy.logger.info(
           `Meetings:index#createMeeting --> Info Unable to fetch meeting info for ${destination}.`
         );
+        if (failOnMissingMeetingInfo) {
+          LoggerProxy.logger.info(
+            `Meetings:index#createMeeting --> Destroying meeting due to missing meeting info.`
+          );
+          // @ts-ignore
+          this.destroy(meeting, MEETING_REMOVED_REASON.MISSING_MEETING_INFO);
+          throw new Error('meeting information not found');
```

Comment:
instead of throwing a generic Error with a message and having to match on the error message later, it would be better to define a new error type e.g. MissingMeetingInfoError (see how PermissionsError is done)